resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -1,14 +1,20 @@
+# Where the puppet-agent dependency places puppet, facter, etc...
+# Puppet server expects to load Puppet from this location
+os-settings: {
+    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
+}
+
 # configuration for the JRuby interpreters
 jruby-puppet: {
     # This setting determines where JRuby will look for gems.  It is also
     # used by the `puppetserver gem` command line tool.
-    gem-home: /var/lib/puppet/jruby-gems
+    gem-home: /opt/puppetlabs/server/data/puppetserver/jruby-gems
 
     # (optional) path to puppet conf dir; if not specified, will use the puppet default
-    master-conf-dir: /etc/puppetlabs/agent
+    master-conf-dir: /etc/puppetlabs/puppet
 
     # (optional) path to puppet var dir; if not specified, will use the puppet default
-    master-var-dir: /opt/puppetlabs/agent/cache
+    master-var-dir: /opt/puppetlabs/server/data/puppetserver
```

Comment:
Makes sense. Generally, if an acceptance test needs to change something in var/confdir, it should create a temp directory and use `with_puppet_running_on` pointing to the temp location. That said I recently found an acceptance test that deleted the puppet user and recreated it!? So yeah, hopefully we don't have any acceptance test modifying puppet.conf to point to an alternate vardir in an ad-hoc manner (not going through beaker's methods for that)..